RecoTracker/IterativeTracking/python/DisplacedGeneralStep_cff.py

```diff
@@ -0,0 +1,243 @@
+    ComponentName = cms.string('displacedGeneralStepChi2Est'),
+    nSigma = cms.double(3.0),
+    MaxChi2 = cms.double(10.0),
+    clusterChargeCut = cms.PSet(refToPSet_ = cms.string('SiStripClusterChargeCutTight')),
+)
+
+
+
+#----------------------------------------- TRACK BUILDING
+import RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi
+displacedGeneralStepTrajectoryBuilder = RecoTracker.CkfPattern.GroupedCkfTrajectoryBuilder_cfi.GroupedCkfTrajectoryBuilder.clone(
+    MeasurementTrackerName = '',
+    trajectoryFilter = cms.PSet(refToPSet_ = cms.string('displacedGeneralStepTrajectoryFilter')),
+    inOutTrajectoryFilter = cms.PSet(refToPSet_ = cms.string('displacedGeneralStepTrajectoryFilterInOut')),
+    useSameTrajFilter = False,
+    minNrOfHitsForRebuild = 4,
+    maxCand = 2,
+    estimator = cms.string('displacedGeneralStepChi2Est'),
+    maxDPhiForLooperReconstruction = cms.double(2.0),
+    # 0.63 GeV is the maximum pT for a charged particle to loop within the 1.1m radius
+    # of the outermost Tracker barrel layer (with B=3.8T)
+    maxPtForLooperReconstruction = cms.double(0.7) 
+    )
+
+
+
+#----------------------------------------- MAKING OF TRACK CANDIDATES
+import RecoTracker.CkfPattern.CkfTrackCandidates_cfi
+displacedGeneralStepTrackCandidates = RecoTracker.CkfPattern.CkfTrackCandidates_cfi.ckfTrackCandidates.clone(
+    src = cms.InputTag('displacedGeneralStepSeeds'),
+    ### these two parameters are relevant only for the CachingSeedCleanerBySharedInput
+    numHitsForSeedCleaner = cms.int32(50),
+    onlyPixelHitsForSeedCleaner = cms.bool(True),
+
+    TrajectoryBuilderPSet = cms.PSet(refToPSet_ = cms.string('displacedGeneralStepTrajectoryBuilder')),
+    clustersToSkip = cms.InputTag('displacedGeneralStepClusters'),
+    doSeedingRegionRebuilding = True,
+    useHitsSplitting = True,
+    cleanTrajectoryAfterInOut = True
+)
+
+from TrackingTools.TrajectoryCleaning.TrajectoryCleanerBySharedHits_cfi import trajectoryCleanerBySharedHits
+displacedGeneralStepTrajectoryCleanerBySharedHits = trajectoryCleanerBySharedHits.clone(
+    ComponentName = cms.string('displacedGeneralStepTrajectoryCleanerBySharedHits'),
+    fractionShared = cms.double(0.25),
+    allowSharedFirstHit = cms.bool(True)
+    )
+displacedGeneralStepTrackCandidates.TrajectoryCleaner = 'displacedGeneralStepTrajectoryCleanerBySharedHits'
+
+
+
+
+
+# ----------------------------------------- TRACK FITTING AND SMOOTHING OPTIONS
+import TrackingTools.TrackFitters.RungeKuttaFitters_cff
+displacedGeneralStepFitterSmoother = TrackingTools.TrackFitters.RungeKuttaFitters_cff.KFFittingSmootherWithOutliersRejectionAndRK.clone(
+    ComponentName = 'displacedGeneralStepFitterSmoother',
+    EstimateCut = 30,
+    MinNumberOfHits = 8,
+    Fitter = cms.string('displacedGeneralStepRKFitter'),
+    Smoother = cms.string('displacedGeneralStepRKSmoother')
+    )
+
+
+displacedGeneralStepFitterSmootherForLoopers = displacedGeneralStepFitterSmoother.clone(
```

Comment:
apparently only the tobTec step has this `*ForLoopers` setup; ostensibly, no looper reconstruction should even be needed for the displacedGeneralStep.
Is there a way to just not have any of these  `*ForLoopers` parts of this config file?